docs/legacy-version-migration.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
I would have called this file... 'migration'. We're certainly talking about migrating from a legacy version.